src/main/java/com/amaze/filemanager/fragments/FTPServerFragment.java

```diff
@@ -496,7 +498,11 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
      * @return address at which server is running
      */
     private String getFTPAddressString() {
-        return "ftp://" + FTPService.getLocalInetAddress(getContext()).getHostAddress() + ":" + FTPService.getPort();
+        String hostAddress = "";
+        InetAddress ia = FTPService.getLocalInetAddress(getContext());
+        if (ia != null)
+            hostAddress = ia.getHostAddress();
+        return "ftp://" + hostAddress  + ":" + FTPService.getPort();
```

Comment:
You are saying that if `ia` is null `getFTPAddressString()` will not return "ftp://:3333"?